src/ListGroup.js

```diff
@@ -21,16 +26,25 @@ class ListGroup extends React.Component {
       });
     }
 
-    if (shouldRenderDiv) {
-      return this.renderDiv(items);
-    }
-    return this.renderUL(items);
+    return shouldRenderDiv ? this.renderDiv(items) : this.renderUL(items);
   }
 
   isAnchorOrButton(props) {
     return (props.href || props.onClick);
   }
 
+  areCustomChildren(children) {
+    let customChildren = false;
+
+    ValidComponentChildren.forEach(children, (child) => {
+      if (child.type.displayName !== 'ListGroupItem') {
```

Comment:
I think we want to assert on this being our `ListGroupItem`, because otherwise we don't know a priori whether it's going to emit a `div` or an `li`.